backend/shared/shared/oidc/auth.py

```diff
@@ -65,7 +66,10 @@ def authenticate(self, request, **kwargs):
         }
 
         # Get the token
-        token_info = self.get_token(token_payload)
+        try:
+            token_info = self.get_token(token_payload)
+        except HTTPError:
```

Comment:
When would this happen? Is it worth logging a warning here?

Reply:
This happens if an user tries to log in with the Azure ADFS login method and Django goes through all the authentication backends. This auth backend is before the ADFS backend so it would try this backend first and raise an error here. I don't think it needs a warning necessarily.